src/NovelRT.DotNet/CMakeLists.txt

```diff
@@ -35,20 +35,20 @@ add_custom_command(
   OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/${OUTPUT_ASSEMBLY}
   BYPRODUCTS ${OUTPUT_BYPRODUCTS}
   COMMAND ${Dotnet_PROGRAM} msbuild ${CSPROJ} ${OPTIONS}
-    /t:Build,Publish
+  /t:Build,Publish
   COMMENT "Building .NET subproject"
 )
 
 add_custom_target(Dotnet_Build
   DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/${OUTPUT_ASSEMBLY}
-)
+  )
 
 add_executable(Dotnet IMPORTED GLOBAL)
 add_dependencies(Dotnet Dotnet_Build)
 set_target_properties(Dotnet
   PROPERTIES
-    IMPORTED_LOCATION ${CMAKE_CURRENT_BINARY_DIR}/publish/IF_YOU_SEE_THIS_YOU_TRIED_TO_RUN_THE_DOTNET_TARGET
-)
+  IMPORTED_LOCATION ${CMAKE_CURRENT_BINARY_DIR}/publish/IF_YOU_SEE_THIS_YOU_TRIED_TO_RUN_THE_DOTNET_TARGET
+  )
```

Comment:
Okay, there's enough of these spacing changes that it warrants the question - what's with this? Many of the CMake files touched is showing this.

Reply:
CLion being CLion and me not caring enough to change it probably.

Reply:
I should also note many modifications happened over the course of this test, so its no surprise if the spacing is slightly different.

Reply:
Maybe go to CLion settings > Editor > Code Style > CMake and set the "Continuation indent" to 2. Worked for me.

Reply:
I think my settings are overridden by clang-format here.

Reply:
Clang-format does not validate for CMake, so it would make sense if CLion was being CLion. I'm not opposed to this, but I can see this being flagged as a nit in the future. That is all :)

Reply:
I'll give it a go in CLion settings as @jeuxjeux20 suggested first! :)

Reply:
Nevermind, CLion is doing the same """"formatting"""" for me as well.

Reply:
OH WHAT FUN! :DDDD

Reply:
Can't wait for thousands of CODE FORMATTING REVIEWS!!!